### DISCUSSION TITLE
games.py

### DISCUSSION THREAD

**Discussion by sawyerru** [mattdodge/yahoofantasy#20] [Q&A] [answered]
Hey where does the games.py file get created from? It looks like you can only add the game codes at the conclusion of the season... I'm trying to get current league data. Thanks, great work - WAYY better than yahoo documentation.

**Comment by mattdodge** [comment, marked as answer]
The games get added manually by me or whoever feels like making a PR. The SDK should work mid-season though, I just hadn't added the game codes yet, thanks for the reminder!

I just pushed a release (1.2.1) that has the 2021 game codes in there, so if you update the package (`pip install -U yahoofantasy`) you should be able to access your 2021 leagues.
